src/plotting/Plotting.jl

```diff
@@ -0,0 +1,4 @@
+
+using CairoMakie
```

Comment:
I'm not sure this file makes sense. Since you just created some util functions, you can just leave everything in a utils file (i.e. `plotting_utils.jl`).

Reply:
Deleted Plotting.jl file